docs/octant-plugin-installation.md

```diff
@@ -28,19 +28,17 @@ your environment and preference.
 1. Create a secret that contains your kubeconfig.
 
     ```bash
-    # Change --from-file according to kubeconfig location in your set up.
-    kubectl create secret generic octant-kubeconfig --from-file=/etc/kubernetes/admin.conf -n kube-system
+    # Change file path "/etc/kubernetes/admin.conf" to kubeconfig location in your set up.
+    kubectl create secret generic octant-kubeconfig --from-file=admin.conf=/etc/kubernetes/admin.conf -n kube-system
```

Comment:
I would suggest using `kubectl create secret generic octant-kubeconfig --from-file=admin.conf=<path to kubeconfig file> -n kube-system`

to avoid any confusion as to whether users should change the `admin.conf` key as well

Reply:
Thanks for the suggestion. I have updated accordingly.